doc/docs/master/how-tos/developer-workflow/index.md

```diff
@@ -0,0 +1,32 @@
+# Developer Workflow
+
+In general, the developer workflow for Pachyderm involves adding 
+data to versioned data repositories, creating pipelines to 
+read from those repositories, executing the pipeline's code, and writing the pipeline's output to other data repositories. . 
+Both the data and pipeline can be iterated on independently with Pachyderm
+handling the code execution according to the pipeline specfication.
+The workflow steps are shown below.
+
+![Developer workflow](../../assets/images/d_steps_analysis_pipeline.svg)
+
+## Data Workflow - Load Your Data into Pachyderm
+
+You need to add your data to Pachyderm so that your pipeline runs your code
+against it. You can do so by using one of the following methods:
```

Comment:
That's correct. I didn't want to change things too much from the original, since I assumed they already had previous review. How does this sound?

```suggestion
Adding data to Pachyderm is the first step towards building data-driven pipelines. There are multiple ways to add data to a Pachyderm repository:
```